root-drive-science-ios.xcodeproj/project.pbxproj

```diff
@@ -519,14 +519,14 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = Z22DE9QC77;
+				DEVELOPMENT_TEAM = 75UNJS9G48;
 				INFOPLIST_FILE = "root-drive-science-ios/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.joinroot.root-drive-science-ios";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.root-enterprise.dsdemo-ios";
```

Comment:
why are the bundle ids changing?

Reply:
they are globally unique and I figured I'd create a new one with `com.root-enterprise` instead of deleting the old one and recreating it on the new team